### PR TITLE
finished challenge

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/multiBracketValidation/multi-bracket.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+let multiBracketMatching = require('../multiBracketValidation/multi-bracket-validation');
```
Comment (adrianhuebner) [inline javascript/code-challenges/multiBracketValidation/multi-bracket.test.js:3]: Everything looks great, would just like to see some tests here!
